buildtest/cli/__init__.py

```diff
@@ -988,6 +988,24 @@ def config_menu(subparsers, parent_parser):
     compilers = subparsers_config.add_parser(
         "compilers", aliases=["co"], help="Search compilers"
     )
+    # buildtest config profile
+    profile = subparsers_config.add_parser(
+        "profile", help="Query profile from buildtest configuration"
```

Comment:
can you change the command to `profiles` with an `s`. So do the following

```python
    profile = subparsers_config.add_parser(
        "profiles", help="Query profile from buildtest configuration")
```

Note the `s` is for the argument in add_parser not the variable `profile`.

Reply:
done